threader3000.py

```diff
@@ -82,6 +85,13 @@ def threader():
     print("Port scan completed in "+str(total))
     print("-" * 60)
 
+    # print suggested nmap scan with:
+    # -sV service and version detection
+    # -sC common scripts
+    # -o scan.nmap save output to file
+    print("suggested nmap:")
```

Comment:
I will accept code modification here in the following format:

    print("Port scan completed in "+str(total))
    print("-" * 60)
    print("Threader3000 recommends the following Nmap scan:")
    print("-" * 60)
    print("nmap -p{ports} -sV -sC {ip}".format(ports=",".join(discovered_ports), ip=target))
    print("-" * 60)
    print("Press Enter to exit...")